calendar_public_holiday/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2015 2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# Copyright 2020 InitOS Gmbh
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Calendar Holidays Public",
+    "summary": """
+        Manage Public Holidays
+    """,
+    "version": "18.0.1.0.0",
+    "license": "AGPL-3",
+    "category": "HR/Calendar",
+    "author": "Camptocamp," "Odoo Community Association (OCA)",
```

Comment:
I think this module should have the same authors as the original `hr_holidays_public` one (apart from yours).